core/framework/extractors/code_extractor_backup_20250913.py

```
@@ -0,0 +1,193 @@
+                    result['symbol_hash'] = self.tree_sitter.generate_symbol_hash(tree_sitter_data['symbols'])
+
+            # Extract basic metadata (fallback for non-Tree-sitter)
+            result['metadata'] = self._extract_metadata(content, file_path)
+
+            # Merge Tree-sitter metrics into metadata if available
+            if tree_sitter_data.get('metrics'):
+                result['metadata'].update(tree_sitter_data['metrics'])
+
+            return result
+
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)